### DISCUSSION TITLE
Notification categories or possibility of filtering them

### DISCUSSION THREAD

**Discussion by TesloCosmin** [dailydotdev/daily#1222] [Feature Requests]
I find the notifications list too cluttered and hard to follow, especially after 1 or 2 days of inactivity. It would be nice to have it broken down into categories (upvotes, new posts, etc.) or have the possibility to filter the notifications somehow. The goal is to make one able to find a specific notification super easy.

E.g. I was trying to find who upvoted my comment and I needed to scroll and search for it.

**Comment by sabdailydev** [comment]
Hi @TesloCosmin , thanks for the suggestion. It will help provide a more consumable experience for the users (especially those who are creating content, submitting posts, engaging in discussions, squad admins, etc.) . We will consider it.

**Comment by nimrodkra** [comment]
Can you provide an example of another platform that does something similar
to what you requested here? It would be helpful to get a reference to
another existing experience that you love to better understand how you
envision it

**Comment by castarco** [comment]
I also think that the notifications system, as it is, can be quite annoying.

For example, I don't really care about people writing stuff in some of the squads I belong to (while other people do, of course), but I care about new articles coming from certain sources.

Being able to choose what I want to be notified for would be a great improvement.

I imagine at least these three categories:
- starred/favorite sources
- squads activity (the one that I would always have off for myself)
- "friends" (or followed users) activity

**Comment by atkr** [comment]
I totally agree with this suggestion. The notification "tab" has way too many notifications and can seemingly not be configured or filtered at all.

Simply being able to filter by "type of notification" (new post, reply, upvote, etc) would be a great start.

I realize one can use the search bar to filter the notifications based on strings, but it's not very efficient as one has to know what keywords to use to filter the different types of notifications.
